FirebaseAppCheck/Tests/Unit/AppAttestProvider/Storage/FIRAppAttestArtifactStorageTests.m

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <XCTest/XCTest.h>
+
+#import "FBLPromise+Testing.h"
+
+#import "FirebaseAppCheck/Sources/AppAttestProvider/Storage/FIRAppAttestArtifactStorage.h"
+
+@interface FIRAppAttestArtifactStorageTests : XCTestCase
+
+@property(nonatomic) NSString *appName;
+@property(nonatomic) NSString *appID;
+@property(nonatomic) FIRAppAttestArtifactStorage *storage;
+
+@end
+
+@implementation FIRAppAttestArtifactStorageTests
+
+- (void)setUp {
+  [super setUp];
+
+  self.appName = @"FIRAppAttestArtifactStorageTests";
+  self.appID = @"1:100000000000:ios:aaaaaaaaaaaaaaaaaaaaaaaa";
+
+  self.storage = [[FIRAppAttestArtifactStorage alloc] initWithAppName:self.appName
+                                                                appID:self.appID
+                                                          accessGroup:nil];
+}
+
+- (void)tearDown {
+  self.storage = nil;
```

Comment:
Maybe call `[self.storage setArtifact:nil]` before `self.storage = nil` to clean up left over storage from `testSetAndGetArtifact` (so the left over storage doesnt persist between test runs).